bands/models.py

```diff
@@ -136,7 +136,7 @@ def w9_extension(self):
 
     @property
     def status(self):
-        return 'Yes <a href="{}">(view file)</a>'.format(self.w9_url) if self.w9_filename else ''
+        return self.w9_url if self.w9_url else ''
```

Comment:
This line is totally fine the way it is, but I'm going to take this opportunity to point out an interesting feature of our codebase that I just learned myself!

If you take any model property and append the suffix `_label`, it will do exactly the same thing. So you could do:
```python
return self.w9_url_label
```

I had no idea!